contracts/gateway-v1.yaml

```diff
@@ -3579,4 +3698,75 @@ components:
         resp:
           $ref: '#/components/schemas/RequestReference'
       xml:
-        name: PatientIdentificationResponse
\ No newline at end of file
+        name: PatientIdentificationResponse
+    HipInitiatedLinkRequest:
```

Comment:
We also need a HIP element, right? because to link we need the HIP Id as well.

Reply:
yeah will add it